aspnetcore/host-and-deploy/iis/modules.md

```diff
@@ -116,6 +116,26 @@ If opting to remove a module with a setting in *web.config*, unlock the module a
     </system.webServer>
    </configuration>
    ```
+   
+In order to manipulate modules for IIS Express using *web.config*, you must modify *applicationHost.config* to unlock the `<modules>` section:
+
+1. Navigate to *{APPLICATION ROOT}\\.vs\config* and open the *applicationhost.config* file.
```

Comment:
missing a double `\`
Why not use / ? Folks know what you mean. Might not copy/paste right on windows tho.

Reply:
Only one is required for the . (period). Without a double there, it thinks you're escaping the period, and a slash doesn't render.